lms/djangoapps/courseware/views/views.py

```diff
@@ -283,8 +283,9 @@ def course_info(request, course_id):
             # redirect to the dashboard page.
             if isinstance(access_response, StartDateError):
                 start_date = strftime_localized(course.start, 'SHORT_DATE')
```

Comment:
@adampalay actual problem is caused by `strftime_localized` which is not returning encoded `start_date`. Using `QueryDict` will make start using default encoding i.e utf-8 in `settings` module of app, which basically according to my understanding just playing encoding part here.  I am fine with your implementation, but my suggestion is that it would be great if we can make `strftime_localized` to work in way that we don't have to explicitly change encoding of a string. We may not know in future which other part of our code will use `strftime_localized` method and will expect an encoded return.